api/core/model_runtime/model_providers/leptonai/leptonai.yaml

```diff
@@ -0,0 +1,29 @@
+provider: leptonai
+label:
+  zh_Hans: Lepton AI
+  en_US: Lepton AI
+icon_small:
+  en_US: icon_s_en.png
+icon_large:
+  en_US: icon_l_en.png
+background: "#F5F5F4"
+help:
+  title:
+    en_US: Get your API Key from Lepton AI
+    zh_Hans: 从 Lepton AI 获取 API Key
+  url:
+    en_US: https://dashboard.lepton.ai/workspace/mm1k227j/settings/api-tokens
```

Comment:
This URL is incorrect because it contains a username, making it non-generic for others.

Reply:
Oops, I am going to fix it. Thank you :)